### ISSUE TITLE
bot runs a few minutes only a few minutes

### ISSUE THREAD

**Issue by tempneff** [viown/Atomic#3]
With default config (accept cookie) the bot runs great for 3-5 minutes then just turns off.  I couldn't find an error log.

**Comment by viown** [comment]
Ran the bot for about an hour, and no issues, it shouldn't crash randomly.

**Comment by Yinthe69** [comment]
It crashed every 3mins for me and now i set it to open as administrator and now it has not crashed yet, so if you have the same problem try to set it run as administrator
https://user-images.githubusercontent.com/78982725/107818213-90a1f100-6d77-11eb-8516-fe52fecdd581.jpg

**Comment by tempneff** [comment]
Yeah, that helps but it still only runs 5-10 minutes for me.  I wonder if it's Windows Defender interrupting.  
In a side question: How does the bot find users to trade with?

**Comment by viown** [comment]
Through groups and limited item resellers, sometimes people from your trade history as well.
